ovos_utils/json_helper.py

```diff
@@ -23,10 +23,12 @@ def merge_dict(base, delta, merge_lists=False, skip_empty=False, no_dupes=True):
                 # dont replace if new entry is empty
                 pass
             elif all((isinstance(b, list), isinstance(d, list), merge_lists)):
-                base[k] += d
                 if no_dupes:
-                    # deduplicate
-                    base[k] = list(set(base[k]))
+                    for thing in d:
```

Comment:
what about a one liner
```python
base[k] += [thing for thing in d if thing not in base[k]]
```

Reply:
That's what I thought, and it should work, but some condition kept making it attempt to iterate over int members of `d`. I can't seem to replicate it now, so it may just have been a failure to properly deepcopy, maybe replacing some list members with actual ints. Not sure.

No problem making it a one liner now.